server/src/main/java/org/opensearch/cluster/metadata/MetadataCreateIndexService.java

```diff
@@ -940,7 +940,7 @@ static Settings aggregateIndexSettings(
      * @param clusterSettings cluster level settings
      * @param combinedTemplateSettings combined template settings which satisfy the index
      */
-    private static void updateReplicationStrategy(
+    public static void updateReplicationStrategy(
```

Comment:
why have we made this public? this is not being used outside this class? or you were intending to reuse this method elsewhere?

Reply:
reusing this method now directly instead of adding another method in RestoreService.